rust/ql/lib/codeql/rust/dataflow/internal/Node.qll

```diff
@@ -472,6 +472,11 @@ newtype TNode =
         any(PrefixExprCfgNode pe | pe.getOperatorName() = "*").getExpr(),
         any(AwaitExprCfgNode a).getExpr(), any(MethodCallExprCfgNode mc).getReceiver()
       ]
+    or
+    exists(ExprCfgNode pred |
+      exists(TExprPostUpdateNode(pred)) and
+      e = getPostUpdateReverseStep(pred, _)
+    )
```

Comment:
We use a `node.getPreUpdateNode().asExpr()` pattern quite a bit. Could that be used here like this?
```ql
e = getPostUpdateReverseStep(any(PostUpdateNode n).getPreUpdateNode().asExpr(), _)
```
That would also look more like the above cases and could be included in the list.